apps/.storybook/preview-head.html

```diff
@@ -3,7 +3,6 @@
 <link rel="stylesheet" media="all" href="css/levelbuilder.css" />
 <link rel="stylesheet" media="all" href="css/pd.css" />
 <link rel="stylesheet" media="all" href="css/droplet/droplet.min.css" />
-<link rel="stylesheet" media="all" href="/css/brstrap.css" />
```

Comment:
Why is this no longer needed?

Reply:
It's related to the removal of `../../pegasus/sites.v3/code.org/public` as one of the static directories, there are a couple of links in the description to relevant PRs that have more detail. tl;dr is that I think it was added to support a component in Storybook that is no longer in Storybook.